python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cv.py

```diff
@@ -271,12 +272,78 @@ def compute_R2_score_by_kfold(
     return r2Score
 
 
+# %%
+# The next function computes the R2 score by K-Fold.
+
+
+def compute_R2_score_by_kfold(
+    inputSample,
+    outputSample,
+    multivariateBasis,
+    totalDegree,
+    distribution,
+    kParameter=5,
+):
```

Comment:
Oups. The next code seems to be duplicated: the same function already exists a few lines upper in the same Python script. I suggest to remove it.

```suggestion
```